app/v1/controllers/requestParsers/validators/SubmitReturnValidator.scala

```diff
@@ -0,0 +1,143 @@
+      JsonFormatValidation.validate[Boolean]((body \ "finalised").get, UnMappedPlayRuleError)
+    )
+  }
+
+  private def mandatoryFieldValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val body = data.body.asJson.get
+
+    List(
+      MandatoryFieldValidation.validate(body \ "periodKey", "periodKey"),
+      MandatoryFieldValidation.validate(body \ "vatDueSales", "vatDueSales"),
+      MandatoryFieldValidation.validate(body \ "vatDueAcquisitions", "vatDueAcquisitions"),
+      MandatoryFieldValidation.validate(body \ "totalVatDue", "totalVatDue"),
+      MandatoryFieldValidation.validate(body \ "vatReclaimedCurrPeriod", "vatReclaimedCurrPeriod"),
+      MandatoryFieldValidation.validate(body \ "netVatDue", "netVatDue"),
+      MandatoryFieldValidation.validate(body \ "totalValueSalesExVAT", "totalValueSalesExVAT"),
+      MandatoryFieldValidation.validate(body \ "totalValuePurchasesExVAT", "totalValuePurchasesExVAT"),
+      MandatoryFieldValidation.validate(body \ "totalValueGoodsSuppliedExVAT", "totalValueGoodsSuppliedExVAT"),
+      MandatoryFieldValidation.validate(body \ "totalAcquisitionsExVAT", "totalAcquisitionsExVAT"),
+      MandatoryFieldValidation.validate(body \ "finalised", "finalised")
+    )
+  }
+
+  private def tierFourValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+
+    val body = data.body.asJson.get.as[SubmitRequestBody]
+
+    List(
+      VATNetValueValidation.validate(body.totalVatDue, body.vatReclaimedCurrPeriod, body.netVatDue),
+      VATTotalValueValidation.validate(body.vatDueSales, body.vatDueAcquisitions, body.totalVatDue)
+    )
+  }
+
+  private def vrnFormatValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    List(
+      VrnValidation.validate(data.vrn)
+    )
+  }
+
+  private def monetaryFormatValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val body: SubmitRequestBody = data.body.asJson.get.as[SubmitRequestBody]
+
+    List(
+      MonetaryValueFormatValidation.validate(body.vatDueSales, "vatDueSales"),
+      MonetaryValueFormatValidation.validate(body.vatDueAcquisitions, "vatDueAcquisitions"),
+      MonetaryValueFormatValidation.validate(body.totalVatDue, "totalVatDue"),
+      MonetaryValueFormatValidation.validate(body.vatReclaimedCurrPeriod, "vatReclaimedCurrPeriod"),
+      MonetaryValueFormatValidation.validate(body.netVatDue, "netVatDue"),
+      MonetaryValueFormatValidation.validate(body.totalValueSalesExVAT, "totalValueSalesExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalValuePurchasesExVAT, "totalValuePurchasesExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalValueGoodsSuppliedExVAT, "totalValueGoodsSuppliedExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalAcquisitionsExVAT, "totalAcquisitionsExVAT"),
```

Comment:
**totalValueSalesExVAT**, ** totalValuePurchasesExVAT**, ** totalValueGoodsSuppliedExVAT** and ** totalAcquisitionsExVAT** are allowed without decimals